crates/core/machine/src/riscv/shape.rs

```diff
@@ -703,7 +703,7 @@ impl<F: PrimeField32> Default for CoreShapeConfig<F> {
     }
 }
 
-#[cfg(any(test, feature = "programs"))]
+#[cfg(test)]
```

Comment:
Why did we turn this off for the programs feature?

Reply:
Oh i see the programs feature is replaced with the test artifacts crate?

Reply:
Yes exactly